environment.yml

```diff
@@ -12,6 +12,7 @@
    - sphinx
    - sphinx-gallery
    - sphinx_rtd_theme
+   - sphinx_markdown_tables
```

Comment:
sphinx_markdown_tables isn't available from conda-forge. You'll need to tell it to install with pip. Replace this line by adding at the end of the file:
```
    - pip:
        - sphinx-markdown-tables
```

Note the PyPI package name uses dashes, not underscores.